ddtrace/appsec/_iast/_patch.py

```diff
@@ -1,8 +1,10 @@
+import functools
 import sys
 from typing import Callable
 from typing import Text
 
 from wrapt import FunctionWrapper
+from wrapt import wrap_function_wrapper as _w
```

Comment:
Why not use our own `try_wrap_function_wrapper` defined in line 37 of this file?

It's the same function but with a try/except for unexpected error handling

Reply:
:+1: